Create ch_knet.m3u

Dum4G opened the pull request:
Resolves https://github.com/iptv-org/iptv/issues/15731
Resolves https://github.com/iptv-org/iptv/issues/15730
Resolves https://github.com/iptv-org/iptv/issues/15729
Resolves https://github.com/iptv-org/iptv/issues/15728
Resolves https://github.com/iptv-org/iptv/issues/15727
Resolves https://github.com/iptv-org/iptv/issues/15726

Dum4G committed:
Create fr_ktvzone.m3u8

Dum4G committed:
Update fr.m3u

Dum4G committed:
Update uk.m3u

BellezaEmporium reviewed (commented):
Will add more channels in the lot.

Best would be renaming this to ch_knet.m3u

BellezaEmporium committed:
Update fr_ktvzone.m3u8

Added a few channels

Rai 2 and 3 exist respectfully at 131 and 132, though they show a banner saying there's a software update that's needed.

BellezaEmporium commented on streams/fr_ktvzone.m3u8:
```diff
@@ -0,0 +1,125 @@
+http://livetv.ktv.zone/4/play.m3u8
+#EXTINF:-1 tvg-id="France3.fr",France 3
+http://livetv.ktv.zone/5/play.m3u8
+#EXTINF:-1 tvg-id="CanalPlus.fr",Canal+
```
```suggestion
#EXTINF:-1 tvg-id="CanalPlus.fr",Canal+ (en clair)
```

Dum4G committed:
Update streams/fr_ktvzone.m3u8

Co-authored-by: Ismaël Moret <lanesh4d0w@gmail.com>

Dum4G committed:
Rename fr_ktvzone.m3u8 to ch_knet.m3u

BellezaEmporium reviewed (approved):
LGTM
